src/deepsparse/server/system_logging.py

```diff
@@ -15,9 +15,42 @@
 from typing import Any
 
 from deepsparse import Pipeline
+from deepsparse.loggers import BaseLogger, MetricCategories
+from fastapi import FastAPI, Request, Response
+from starlette.middleware.base import BaseHTTPMiddleware
 
 
-__all__ = ["log_resource_utilization", "log_request_details"]
+__all__ = ["log_resource_utilization", "log_request_details", "SystemLoggingMiddleware"]
+
+REQUEST_DETAILS_IDENTIFIER_PREFIX = "request_details"
```

Comment:
should move this to top level helper file so pipelines can also reference this constant.